### ISSUE TITLE
pytorch 1.3 error

### ISSUE THREAD

**Issue by jinfagang** [open-mmlab/mmcv#136]
```
    model = init_detector(args.config, args.checkpoint, device="cpu")
  File "/usr/local/lib/python3.5/dist-packages/mmdet-1.0rc0+aab283c-py3.5-linux-x86_64.egg/mmdet/apis/inference.py", line 36, in init_detector
    checkpoint = load_checkpoint(model, checkpoint)
  File "/usr/local/lib/python3.5/dist-packages/mmcv/runner/checkpoint.py", line 188, in load_checkpoint
    load_state_dict(model, state_dict, strict, logger)
  File "/usr/local/lib/python3.5/dist-packages/mmcv/runner/checkpoint.py", line 96, in load_state_dict
    rank, _ = get_dist_info()
  File "/usr/local/lib/python3.5/dist-packages/mmcv/runner/utils.py", line 21, in get_dist_info
    initialized = dist.is_initialized()
AttributeError: module 'torch.distributed' has no attribute 'is_initialized'

```

Any idea?

**Comment by hellock** [comment]
The reason may be that `torch.distributed.is_available()` is False, so that `is_initialized` is not imported.

Ref: https://github.com/pytorch/pytorch/blob/master/torch/distributed/__init__.py#L16-L17

**Comment by hellock** [comment]
Fixed in #128
